sdk/metricsadvisor/azure-ai-metricsadvisor/azure/ai/metricsadvisor/models/_models.py

```diff
@@ -1522,41 +1610,45 @@ def _to_generated_patch(self):
 class AzureLogAnalyticsDataFeedSource(object):
     """AzureLogAnalyticsDataFeedSource.
 
-    :param tenant_id: Required. The tenant id of service principal that have access to this Log
+    :keyword str tenant_id: The tenant id of service principal that have access to this Log
```

Comment:
Is it the case for these data feed source models that you can pass a credential entity OR the other cred params? If so, should we include a snippet saying that in the docs? I'm not sure if it's immediately clear.